src/implementations/vanilla/components/gallery/gallery.js

```diff
@@ -553,6 +556,7 @@ export class Gallery {
     );
     if (shareHref != null) {
       this.overlayShare.href = shareHref;
+      this.overlayShare.setAttribute('aria-describedby', `${id}-title`);
```

Comment:
we can assume that an id will be always set when using the template we provide, but before using it i would still check if its not false